core/kubernetes/main/k8s.sh

```diff
@@ -3,6 +3,14 @@
 if [ "$1" == "up" ]; then
     kubectl apply -f ./openhim
     kubectl apply -f ./hapi-fhir
+
+    if [ "$2" == "dev" ]; then
+        kubectl apply -f ../dev/openhim
+        kubectl apply -f ../dev/hapi-fhir
+        
+        echo -e "\nCurrently in development mode\n"
+    fi
+
```

Comment:
If we create the bash script for the dev setup then this can be removed as this will be included in the `setup.dev.sh` script